src/input/Search.js

```diff
@@ -106,8 +107,8 @@ Search.propTypes = {
   showLoadingIcon: PropTypes.bool,
   loadingIcon: PropTypes.object,
   clearIcon: PropTypes.oneOf(PropTypes.object, PropTypes.bool),
-  textInputRef: PropTypes.string,
-  containerRef: PropTypes.string,
+  textInputRef: PropTypes.func,
```

Comment:
You might wanna consider using `PropTypes. oneOfType([PropTypes.func, PropTypes.string])` to keep it backwards compatible. Valid for all other ref replacements.